### PR TITLE
Update default POD CIDR 10.99.0.0/20 & made it configurable

### LINKED ISSUE #39: Change default PID network to prevent collisions
User issue reported:

```
network 10.32.0.0/12 overlaps with existing route 10.47.159.0/24 on host
```

Potential workaround: https://github.com/weaveworks/weave/issues/2736#issuecomment-453083678

PR and test with `10.99.0.0/20` also update limits (number of functions), architecture and FAQ docs on this setting. Make it advanced setting in OVF (end of deployment options setting)?

cc/ @kremerpatrick

### PR DESCRIPTION
**Pull request by lamw**
Fixes #39 

Build:
- [x] Successfully built and deploy VEBA Appliance

Test:
- [x] Deployed VEBA using defaults and confirm weave network is using 10.99.0.0/20
- [x] Deployed VEBA and customized POD CIDR Network to use 10.100.0.0/20 and confirm weave network is using the specified network

Signed-off-by: William Lam <wlam@vmware.com>

### CODE REVIEW

File: manual/photon-dev.xml.template [hunk 1]
```diff
@@ -30,6 +30,10 @@
             <Label>DNS Domain</Label>
             <Description>DNS Domain</Description>
         </Property>
+        <Property ovf:key="guestinfo.ntp" ovf:type="string" ovf:userConfigurable="true" ovf:value="pool.ntp.org">
```
Comment (embano1) [inline manual/photon-dev.xml.template]: How did that get in here? Looks like this could cause conflicts with the ntp PR.

File: photon-dev.json [hunk 2]
```diff
@@ -12,14 +12,16 @@
     "ovftool_deploy_vm_prefix": "24",
     "ovftool_deploy_vm_gateway": "192.168.30.1",
     "ovftool_deploy_vm_dns": "192.168.30.1",
+    "ovftool_deploy_vm_ntp": "pool.ntp.org",
```
Comment (embano1) [inline photon-dev.json]: see comment above, should not be in this commit

File: photon-dev.json [hunk 3]
```diff
@@ -12,14 +12,16 @@
...
     "ovftool_deploy_vm_dns_domain": "primp-industries.com",
     "ovftool_deploy_vm_root_password": "VMware1!",
     "ovftool_deploy_vm_openfaas_password": "VMware1!",
     "ovftool_deploy_vm_vcenter_server": "192.168.30.200",
     "ovftool_deploy_vm_vcenter_username": "administrator@vsphere.local",
     "ovftool_deploy_vm_vcenter_password": "VMware1!",
     "ovftool_deploy_vm_network": "VM Network",
-    "ovftool_deploy_vm_datastore": "sm-vsanDatastore"
+    "ovftool_deploy_vm_datastore": "sm-vsanDatastore",
```
Comment (embano1) [inline photon-dev.json]: ???

File: manual/photon-dev.xml.template (hunk 1 above)
Comment (lamw) [reply manual/photon-dev.xml.template]: This is part of the "dev" template that I use to test deployments. Doesn't affect the actual build

**Pull request merged**